modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-updating-cluster-manifests_{context}"]
+= Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+
+// TODO: what is the right title? Can we find anything more generic than calling out the two resources?
+// Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+// Updating the cluster with the mirror registry manifests
+// Updating the ICSPs and catalog sources in the cluster
+// ???
+
+// TODO: intro
+
+.Prerequisites
+
+* You have mirrored the image set to the registry mirror in the disconnected environment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the OpenShift CLI as a user with the `cluster-admin` role.
+
+. Apply the generated manifests to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f ./oc-mirror-workspace/results-1639608409/
+----
+
+// TODO: Any example output to show?
+
+.Verification
+
+* TODO: Any verification steps?
```

Comment:
@afflom @jpower432 Are there any verification steps that we can show to help users see that this worked?